<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RuleEntry</class>
 <widget class="QWidget" name="RuleEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="visibleCheckBox">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="text">
        <string>Enter a name...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="selectedCheckBox">
       <property name="text">
        <string>Mark for deletion</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="startingRuleLabel">
       <property name="text">
        <string>Starting rule:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="startComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="endRuleLabel">
       <property name="text">
        <string>Rule:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ruleComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="probabilityLabel">
       <property name="text">
        <string>Probability:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="probabliltyValue"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="startLabel">
       <property name="text">
        <string>Rule:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ruleProductionLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="undoButton">
       <property name="text">
        <string>Undo</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
